2019-07-07 22:50:15 jdbc[5]: null
org.h2.message.DbException: The connection was not closed by the application and is garbage collected [90018-197]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.jdbc.JdbcConnection.closeOld(JdbcConnection.java:197)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:137)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:733)
	at org.h2.server.web.WebApp.login(WebApp.java:954)
	at org.h2.server.web.WebApp.process(WebApp.java:211)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:134)
	at org.h2.server.web.WebThread.run(WebThread.java:90)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: The connection was not closed by the application and is garbage collected [90018-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	... 14 more
2019-07-07 22:52:11 jdbc[5]: exception
org.h2.jdbc.JdbcSQLException: Syntax error in SQL statement "DROP BASE[*]"; expected "TABLE, INDEX, USER, SEQUENCE, CONSTANT, TRIGGER, VIEW, ROLE, ALIAS, SCHEMA, ALL, DOMAIN, TYPE, DATATYPE, AGGREGATE, SYNONYM"; SQL statement:
DROP BASE [42001-197]
2019-07-08 14:27:23 database: wrong user or password; user: "SA"
org.h2.message.DbException: Wrong user name or password [28000-197]
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.message.DbException.get(DbException.java:155)
	at org.h2.message.DbException.get(DbException.java:144)
	at org.h2.engine.Engine.openSession(Engine.java:107)
	at org.h2.engine.Engine.openSession(Engine.java:179)
	at org.h2.engine.Engine.createSessionAndValidate(Engine.java:157)
	at org.h2.engine.Engine.createSession(Engine.java:140)
	at org.h2.engine.Engine.createSession(Engine.java:28)
	at org.h2.engine.SessionRemote.connectEmbeddedOrServer(SessionRemote.java:351)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:124)
	at org.h2.jdbc.JdbcConnection.<init>(JdbcConnection.java:103)
	at org.h2.Driver.connect(Driver.java:69)
	at org.h2.server.web.WebServer.getConnection(WebServer.java:733)
	at org.h2.server.web.WebApp.test(WebApp.java:892)
	at org.h2.server.web.WebApp.process(WebApp.java:221)
	at org.h2.server.web.WebApp.processRequest(WebApp.java:170)
	at org.h2.server.web.WebThread.process(WebThread.java:134)
	at org.h2.server.web.WebThread.run(WebThread.java:90)
	at java.lang.Thread.run(Unknown Source)
Caused by: org.h2.jdbc.JdbcSQLException: Wrong user name or password [28000-197]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:357)
	... 19 more
